packages/jats-xml/src/download.ts

```diff
@@ -0,0 +1,37 @@
+import checkdoi from 'doi-utils';
+import fetch from 'node-fetch';
```

Comment:
There might be a bit extra to do here to override fetch to make this work client side.

Reply:
What do you mean here - What wouldn't work?

Reply:
I am not sure if node-fetch is designed for the browser. Will check and see if it fails.